Getting-Started-with-Android-Development.md

```diff
@@ -0,0 +1,20 @@
+# Getting started with Android Development
+
+So, you want to get started with Android development? Let me take you through some basic steps you should go through.
+
+#### Tools
+At first, make sure you have all the tools in your system. The things you'll need to start building application for Android are:
+* [Latest version of the Java JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+* [Android Studio](http://developer.android.com/sdk/index.html)
+* If possible, an Android device
+
+Some basic courses and tutorials for learning Android development:
+* [Udacity course](https://www.udacity.com/course/developing-android-apps--ud853)
+* [Tutorial by developers at Google](http://developer.android.com/training/index.html)
+* [TutorialsPoint](http://www.tutorialspoint.com/android/)
+
+So, what are you waiting for? Just start right away!
+
+For any help, [StackOverflow](stackoverflow.com) is there anyway!
```

Comment:
No, link to Stack Overflow Android tagged answers. It would be good to refer to official Android IRC channel too.